warehouse/templates/pages/help.html

```diff
@@ -83,6 +84,7 @@ <h2><a href="#my-account">My Account</a></h2>
           <li><a href="#compromised-password">{{ compromised_password() }}</a></li>
           <li><a href="#2fa">{{ twoFA() }}</a></li>
           <li><a href="#totp">{{ totp() }}</a></li>
+          <li><a href="#utfkey">{{ utfkey() }}</a></li>
```

Comment:
I did a double-take at `utf` :slightly_smiling_face: 
IIUC this is about U2F keys, not Unicode’s UTF, right?

Reply:
It is about U2F keys and not Unicode's UTF; the T stands for "two". I'm ok with this since it's just an anchor tag, but if you want to change it I figure a PR would be pretty easy!